javascript/javascript1/week2/fullnameFunaction.js

```diff
@@ -0,0 +1,14 @@
+function getFullname(firstname, surname, useFormalName) {
```

Comment:
give `useFormalName` a default value like this
```js
function getFullname(firstname, surname, useFormalName = false) {
```
Then it will be optional and you can write
```js
const fullName2 = getFullname("Dave", "Copper");
```
without passing `useFormalName`